src/quantity.rs

```diff
@@ -210,6 +210,35 @@ macro_rules! quantity {
             impl super::Conversion<V> for super::$unit {})+
         }
 
+        mod str {
+            storage_types! {
+                types: All;
+
+                use $crate::lib::str::FromStr;
+                use $crate::str::ParseQuantityError::*;
+
+                impl<U> FromStr for super::super::$quantity<U, V>
+                where
+                    U: super::super::super::Units<V> + ?Sized,
+                {
+                    type Err = $crate::str::ParseQuantityError;
+                    #[allow(unreachable_patterns)]
```

Comment:
Still needed?

Reply:
Yes, which is somewhat strange (but I think probably just a limitation of the current macros implementation).